source/changelog.txt

```diff
@@ -12,6 +12,22 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v2.1.0
+------
+
+*Released November 21, 2023*
+
+- :issue:`MONGOSH-1621` and :issue:`NODE-5709` – Homebrew users who were automatically upgraded  
+  to Node.js 21 stop seeing deprecation warnings. 
+
+- :issue:`MONGOSH-1452` and :issue:`NODE-5040` – ``mongosh`` now displays BSON objects in a format 
+  that is more consistent with other ``mongosh`` output. Also, many BSON objects now support syntax 
+  highlighting.
+
+- :issue:`MONGOSH-1527` – You can now iterate mongosh cursors with idiomatic 
+  syntax: for (const doc of db.coll.find()) { }. Previously, ``mongosh`` only supported ``.forEach`` 
+  syntax for iteration.
```

Comment:
The `for (const doc of db.coll.find()) { }` should probably be a code block as well?